scalafix/rules/src/main/scala/fix/Collectionstrawman_v0.scala

```diff
@@ -132,13 +136,55 @@ case class Collectionstrawman_v0(index: SemanticdbIndex)
           close <- ctx.matchingParens.close(open.asInstanceOf[Token.LeftBracket])
         } yield
           ctx.replaceToken(open, "(") +
-            ctx.replaceToken(close, ")")).getOrElse(Patch.empty)
+            ctx.replaceToken(close, ")")
+        ).asPatch
+    }.asPatch
+
+  def replaceTupleZipped(ctx: RuleCtx) =
+    ctx.tree.collect {
+      case tupleZipped(Term.Select(Term.Tuple(args), name)) =>
+        val removeTokensPatch =
+          (for {
+            zipped <- name.tokens.headOption
+            closeTuple <- ctx.tokenList.leading(zipped).find(_.is[Token.RightParen])
+            openTuple <- ctx.matchingParens.open(closeTuple.asInstanceOf[Token.RightParen])
+            maybeDot = ctx.tokenList.slice(closeTuple, zipped).find(_.is[Token.Dot])
```

Comment:
Utilities to find dots is missing in scalafix, I've written something similar to this many times 😅